assets/pages/new-contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -132,20 +131,29 @@ function onClick(event, props: PropTypes) {
   }
 }
 
-// Set this to true to enable Google Pay (good for development)
-const weAreSupportingGooglePay = false;
-
 // The value of result will either be:
 // . null - browser has no compatible payment method)
 // . {applePay: true} - applePay is available
-// . {applePay: false} - GooglePay or PaymentRequestApi available
-const browserIsCompatible = (result: Object) => {
-  if (weAreSupportingGooglePay) {
-    return result !== null;
+// . {applePay: false} - GooglePay, Microsoft Pay and PaymentRequestApi available
+const availablePaymentRequestButtonPaymentMethod: Object => (StripePaymentMethod | null) = (result: Object) => {
```

Comment:
don't we normally use the other return type notation?

```javascript
const availablePaymentRequestButtonPaymentMethod = (result: Object): StripePaymentMethod | null =>
...
```

I find this one hard to parse since I'm used to the other one - and it seems you need to notate the `Object` input type twice which I'm not a fan of

Reply:
👍 sorry about this, not sure what came over me...